packages/frontend/src/components/common/modal/ChartCreateModal/SaveToSpaceOrDashboard.tsx

```diff
@@ -312,15 +313,14 @@ export const SaveToSpaceOrDashboard: FC<SaveToSpaceOrDashboardProps> = ({
                     },
                     ...getDefaultChartTileSize(savedData.chartConfig?.type),
                 };
-                await updateDashboard({
-                    name: values.dashboardName
-                        ? values.dashboardName
-                        : 'untitled',
+                const updateFields: DashboardVersionedFields = {
                     filters: selectedDashboard.filters,
```

Comment:
this doesn't include `name` anymore, don't we need it?

Reply:
No, we don't.  That was also part of the issue , we were overriding the name with untitled.